pipelines/ci/check-pr-author-matches-commit-signature.yml

```diff
@@ -0,0 +1,114 @@
+resources:
+- name: gsp-pr
+  type: pull-request
+  source:
+    access_token: ((pr-build-status))
+    disable_forks: true
```

Comment:
this probably should not be set - people opening PRs from forks should also be told if their PR has an approval/signature problem, we don't want it to silently do nothing